hnn_core/pyramidal.py

```diff
@@ -291,6 +291,7 @@ def __init__(self, gid=-1, pos=-1, p={}):
 
         # run record_current_soma(), defined in Cell()
         self.record_current_soma()
+        self.record_voltage_soma()
```

Comment:
why is this added separately for all the cell classes instead of the base class?

Reply:
Mainly to keep consistency with how `record_current_soma()` is implemented. Given the similarity in their implementation I think it's less confusing if the code is grouped like this.

I can change it now, but I'd prefer to hold off until the "record current" PR since that will likely restructure the existing template for storing continuous data.

Reply:
ok, sounds good. Let's keep tab of things you are postponing to the next PR. So we don't forget